test/test_modeling_question_answering.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import pytest
+from math import isclose
+import numpy as np
+
+from haystack.modeling.infer import QAInferencer
+from haystack.modeling.data_handler.inputs import QAInput, Question
+
+
+def test_inference_different_inputs(bert_base_squad2):
+    qa_format_1 = [
+        {
+            "questions": ["Who counted the game among the best ever made?"],
+            "text": "Twilight Princess was released to universal critical acclaim and commercial success. It received perfect scores from major publications such as 1UP.com, Computer and Video Games, Electronic Gaming Monthly, Game Informer, GamesRadar, and GameSpy. On the review aggregators GameRankings and Metacritic, Twilight Princess has average scores of 95% and 95 for the Wii version and scores of 95% and 96 for the GameCube version. GameTrailers in their review called it one of the greatest games ever created."
+        }]
+    q = Question(text="Who counted the game among the best ever made?")
+    qa_format_2 = QAInput(questions=[q],doc_text= "Twilight Princess was released to universal critical acclaim and commercial success. It received perfect scores from major publications such as 1UP.com, Computer and Video Games, Electronic Gaming Monthly, Game Informer, GamesRadar, and GameSpy. On the review aggregators GameRankings and Metacritic, Twilight Princess has average scores of 95% and 95 for the Wii version and scores of 95% and 96 for the GameCube version. GameTrailers in their review called it one of the greatest games ever created.")
+
+
+    result1 = bert_base_squad2.inference_from_dicts(dicts=qa_format_1)
+    result2 = bert_base_squad2.inference_from_objects(objects=[qa_format_2])
+    assert result1 == result2
+
+
+@pytest.fixture()
```

Comment:
lets cleanup the tests once we are at it (lets not cleanup modelling code so much for now).

So I think fixtures should go into conftest.py and if you see other opportunities for simpler or cleaner code please go for it.

Reply:
Let's talk about that. I think it's similar to the reasoning behind the util functions: "...if they are only used inside a single script I moved them to the script..."
The two test fixtures are "no_answer_inference_result" and "span_inference_result". I would like to better understand if we might use them in other test modules or only in test_modeling:question_answering.py